api/mailer/Util.js

```diff
@@ -0,0 +1,69 @@
+const axios = require('axios');
```

Comment:
the functions in here have to be a part of the auth API. mailer will be running in a container that doesnt have access to mognodb so this wouldnt work, see below

![image](https://media.discordapp.net/attachments/693698635988336691/724767760441278484/SCE_Website_Architecture_2.png?width=954&height=666)

Reply:
So I should move this into `api/routes/Auth.js`?